build/Dockerfile

```diff
@@ -0,0 +1,45 @@
+
+FROM debian:buster
+
+RUN apt-get update && \
+    apt-get upgrade -y && \
+    apt-get install -y --no-install-recommends curl python-pygments git ca-certificates && \
+    apt-get install -y wget && \
+    rm -rf /var/lib/apt/lists/*
+
+RUN curl -sL https://deb.nodesource.com/setup_10.x | bash -
+
+RUN curl -sS https://dl.yarnpkg.com/debian/pubkey.gpg | apt-key add - && \
+    echo "deb http://dl.yarnpkg.com/debian/ stable main" | tee /etc/apt/sources.list.d/yarn.list
+
+RUN apt-get update && \
+    apt-get install -y --no-install-recommends nodejs yarn && \
+    apt-get clean && \
+    rm -rf /var/lib/apt/lists/*
+RUN yarn global add gulp
+WORKDIR /tmp
+COPY ./package.json /tmp/package.json
+COPY ./yarn.lock /tmp/yarn.lock
+
+RUN yarn install && \
+    mkdir -p /opt/clearhaus/gateway-api-docs && \
+    cp -a /tmp/node_modules /opt/clearhaus/gateway-api-docs
+RUN yarn
+RUN curl -s https://api.github.com/repos/gohugoio/hugo/releases \
+    | grep  browser_download_url \
+    | grep Linux-64bit.deb \
+    | grep -v extended \
+    | head -1 \
+    | cut -d '"' -f 4 \
+    | wget -i -
+
+RUN dpkg -i hugo*_Linux-64bit.deb
```

Comment:
Another idea because I couldn't stop myself:

```
RUN URL=$(curl -sS https://api.github.com/repos/gohugoio/hugo/releases | egrep -o 'https://[^"]+/hugo_[0-9.]+_Linux-64bit.deb' | head -n 1) && \
    FILENAME=$(basename "$URL") && \
    curl -sS "$URL" > "$FILENAME" && \
    dpkg -i "$FILENAME" && \
    rm "$FILENAME"
```

Feel free to ignore it.

Reply:
The idea is not tested, btw. I don't know if it's ok to write `MY_VAR=$(some command)` in `RUN` statements in a Dockerfile.

Reply:
I snatched it from nonPCI.
I see https://linux-packages.com/debian-10-buster/package/hugo includes the Hugo package.